libs/filamat/src/UibGenerator.cpp

```diff
@@ -45,13 +45,15 @@ UniformInterfaceBlock const& UibGenerator::getPerViewUib() noexcept  {
             // view
             .add("resolution",              1, UniformInterfaceBlock::Type::FLOAT4, Precision::HIGH)
             // camera
+            .add("cameraForward",           1, UniformInterfaceBlock::Type::FLOAT3, Precision::HIGH)
+            .add("padding0",             1, UniformInterfaceBlock::Type::FLOAT)
```

Comment:
nit^99: indentation is off

Reply:
40fde81a0cc15959dd613d9b8f46d35f12b91f68